packages/react-composites/src/composites/ChatComposite/file-sharing/FileUploadButton.tsx

```diff
@@ -47,6 +47,16 @@ export const FileUploadButton = (props: FileUploadButtonProps): JSX.Element => {
     }
   });
 
+  const iconButtonClassName = mergeStyles({
+    color: 'unset',
+    width: '1.5rem',
+    height: '1.5rem',
+    ':hover': {
```

Comment:
when we are hovering the buttons do we want no change in appearance? what do the other buttons on the control bar do?

Reply:
> overall looks good! the only thing that I am wondering right now is there is a series of tests where the snapshot suddenly has the more options button showing on the message.
> ![image](https://user-images.githubusercontent.com/94866715/169345184-2a8588f5-7027-4266-b2ad-22ce1910610d.png)
> Do we know whats going on here?

This is the issue related to flakiness of tests, where snapshot is taken before the users appear or mouse click is triggered, will be handling this in a separate PR.

Reply:
> when we are hovering the buttons do we want no change in appearance? what do the other buttons on the control bar do?

Icon button adds the color and background when you hover on the icon, we are trying to keep it close to the previous styles so overriding the hovering styles.